helpdesk/teams/forms.py

```diff
@@ -0,0 +1,13 @@
+from django import forms
+
+from teams.models import Team, TeamAttendanceEvent, TeamAttendanceEventType
+
+
+class TeamAttendanceLogForm(forms.ModelForm):
+    type = forms.Select(choices=TeamAttendanceEventType.choices)
+    comment = forms.CharField(required=False)
```

Comment:
Issue: These should be the default types for these fields. You should be able to remove these lines entirely.